src/components/table-content-wrapper/table-content-wrapper.jsx

```diff
@@ -21,7 +21,8 @@ class TableContentWrapper extends Component {
   render() {
     const {
       classes,
-      captionHidden,
+      captionVisibleMobile,
```

Comment:
Is there a possibility to send in `captionVisible = true` as shortcase, and the option for `captionVisible = { mobile: true }` if you want only one visible? That would result in less typing/props without that much extra complexity.